src/redux/reducers.js

```diff
@@ -1,18 +1,22 @@
-import { SET_AUTH_STATE, AuthStates } from './actions.js';
+import { AuthActionTypes, PermissionStates } from './actions.js';
 
-const authState = (state = AuthStates.GUEST, action) => {
+const permissions = (state = PermissionStates.GUEST, action) => {
     switch(action.type) {
-        case SET_AUTH_STATE:
-            return action.authState;
+        case AuthActionTypes.GUEST:
+            return PermissionStates.GUEST;
```

Comment:
I'll explain it and you can tell me if you think I'm wrong. Basically, AuthStates is the interface for the database. As far as I know, the database works off of role-based permissions, which doesn't match up with the front end. I want to be able to handle any changes in the values of the role field on the database easily. AuthStates lets me change those values app-wide by altering one key-value pair. AuthActionTypes is just a convenient way of storing the different types of authentication actions used by the Redux store. PermissionStates holds the different permissions assigned to each user type so that I can update the Redux store conveniently, or compare them to the current user's permissions.